orchagent/pfcwdorch.h

```diff
@@ -55,11 +56,12 @@ class PfcWdOrch: public Orch
     void createEntry(const string& key, const vector<FieldValueTuple>& data);
     void deleteEntry(const string& name);
     PfcFrameCounters getPfcFrameCounters(sai_object_id_t portId);
+    QueueMcDropCounters getQueueMcDropCounters(const Port& port, uint8_t pfcMask);
 
     shared_ptr<DBConnector> m_countersDb = nullptr;
     shared_ptr<Table> m_countersTable = nullptr;
 
-    map<sai_object_id_t, PfcFrameCounters> m_pfcFrameCountersMap;
+    map<sai_object_id_t, pair<PfcFrameCounters, QueueMcDropCounters>> m_pfcFrameCountersMap;
```

Comment:
consider change the name of m_pfcFrameCountersMap since this is not only map of PfcFrameCounters now.

Reply:
Done